README.md

```diff
@@ -26,12 +26,25 @@ To build for Windows you will need to have CMake installed on your system.
  ```
  ```
  # Not specifying -G will automatically generate 32-bit projects.
+ # This will generate a 64bits build with VS2017 compiler
  cmake .. -G "Visual Studio 15 2017 Win64" -DCMAKE_PREFIX_PATH="C:\Qt\5.10.1\msvc2017_64" -DUSE_QT=YES
+
+ # This will generate a 64bits build with VS2019 compiler
+  cmake .. -G "Visual Studio 16 2019" -DCMAKE_PREFIX_PATH="C:\Qt\5.15.2\msvc2019_64" -DUSE_QT=YES
```

Comment:
starting from VS 16 2019, you have to specify arch using `-A x64` or `-A Win32`, and i do believe it defualts to 32 bits if you dont specify

Reply:
On my side it seems to be compiling in `x64`, perhaps because I specified the `msvc2019_64` folder.